.github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml

```diff
@@ -132,7 +132,7 @@ jobs:
 
           $params = @{
                   schoolYear               = "2022"
-                  testSets                 = @("NonDestructiveApi", "NonDestructiveSdk", "DestructiveSdk")
```

Comment:
Disabled the destructive smoke tests because we need to bring [ODS-5432](https://tracker.ed-fi.org/browse/ODS-5432) for them to succeed.

Reply:
Is there a ticket created that includes enabling the destructive smoke tests?

Reply:
No, per @vimayya there are no plans (yet) to bring these fixes down to the 5v line.